go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	knative.dev/eventing v0.27.2
 	knative.dev/hack v0.0.0-20220216040439-0456e8bf6547
 	knative.dev/kn-plugin-event v1.0.1
-	knative.dev/kn-plugin-func v0.21.0
+	knative.dev/kn-plugin-func v0.22.1
```

Comment:
@lance I think we have had wrong display version for that release, the content seemed to be the right one.

Reply:
Yes, the upstream is one minor version ahead of our serverless version numbers so it's easy to get confused.